app/src/main/AndroidManifest.xml

```diff
@@ -22,6 +22,27 @@
     <activity
       android:name=".home.HomeActivity"
       android:screenOrientation="portrait" />
+    <activity
+      android:name=".option.OptionActivity"
+      android:screenOrientation="portrait">
+      <intent-filter>
```

Comment:
Please do not make this as launcher activity. I understand that this activity currently does not have a parent but in such cases, you should write instructions in the PR description about how to run this feature. Revert this launcher code.

Reply:
Reverted.